### ISSUE TITLE
Max int64 not downloaded correctly

### ISSUE THREAD

**Issue by rvernica** [Paradigm4/SciDB-Py#98]
If an array contains an attribute of type `int64` and the value is the maximum permissible positive value, the value is not correctly downloaded by the `scidbpy`.

In the following example, an array is created and a maximum permissible positive value is stored in the array. When the array is downloaded back, the value is different:
```python
>>> import scidbpy
>>> db = scidbpy.connect()
>>> db.afl.create_array('foo', '<x:int64>[i=0:0]', False).eval(store=False)
>>> x = 2**(64 - 1) - 1
>>> x
9223372036854775807L
>>> db.afl.build('foo', "'[({})]'".format(x), True).store('foo').eval(store=False)
>>> db.wrap_array('foo').toarray()
array([-9223372036854775808])
>>> db.wrap_array('foo').todataframe()
                     x
i                     
0 -9223372036854775808
>>> db.wrap_array('foo').tosparse()
array([(0L, -9223372036854775808L)], 
      dtype=[('i', '<i8'), ('x', '<i8')])
>>> db.wrap_array('foo').toarray().dtype
dtype('int64')
```
When checking with `iquery`, the correct value is stored in the array:
```bash
AFL% scan(foo);
{i} x
{0} 9223372036854775807
```
---
* SciDB `16.9`
* SciDB-Py [0afda10](https://github.com/Paradigm4/SciDB-Py/tree/0afda10aa7987a993496668094cdb68cd4141321)

**Comment by rvernica** [comment]
A similar behavior is observed for `uint64` as well:
```python
>>> db.afl.create_array('bar', '<y:uint64>[i=0:0]', False).eval(store=False)
>>> y = 2**64 - 1
>>> y
18446744073709551615L
>>> db.afl.build('bar', "'[({})]'".format(y), True).store('bar').eval(store=False)
>>> db.wrap_array('bar').toarray()
array([0], dtype=uint64)
>>> db.wrap_array('bar').todataframe()
   y
i   
0  0
>>> db.wrap_array('bar').tosparse()
array([(0L, 0L)], 
      dtype=[('i', '<i8'), ('y', '<u8')])
```
```bash
AFL% scan(bar);
{i} y
{0} 18446744073709551615
```

**Comment by rvernica** [comment]
Fixed in SciDB-Py `16.9.1`.
